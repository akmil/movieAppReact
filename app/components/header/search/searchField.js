import React from 'react';

export const SearchField = ({val}) => {
    return (
        <div className={'search-field-box'}>
            <label className={'search-field-label'}>Find your movie</label>
            <input placeholder={`Hey, ${val}`} className={'search-field-input'}/>
        </div>
    );
};

