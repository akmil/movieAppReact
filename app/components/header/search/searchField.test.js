import React from 'react';
// import { shallow } from 'enzyme';
import {SearchField} from './searchField';
import renderer from "react-test-renderer";


describe('<search SearchField/>', () => {
        it('should render <SearchField /> components', () => {
            // const wrapper = shallow(<SearchFilter />);
            const wrapper = renderer.create(<SearchField/>);
            expect(wrapper).toMatchSnapshot();
        });
    }
);

