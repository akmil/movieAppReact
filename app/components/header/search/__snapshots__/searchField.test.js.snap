// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<search SearchField/> should render <SearchField /> components 1`] = `
<div
  className="search-field-box"
>
  <label
    className="search-field-label"
  >
    Find your movie
  </label>
  <input
    className="search-field-input"
    onChange={undefined}
    placeholder="Hey"
  />
</div>
`;
